mainnet-1/specs/evmos.json

```diff
@@ -942,43 +942,6 @@
                             "type": "POST",
                             "add_on": "debug"
                         },
-                            "apis": [
```

Comment:
these api needs to be disabled, they are inherited from ETH and are not available on evmos